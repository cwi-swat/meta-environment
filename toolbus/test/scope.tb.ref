line 7: double declaration of let variable `R'
line 7: (warning) variable `R' redefines let variable in enclosing scope
line 7: (warning) variable `R' redefines let variable in enclosing scope
line 7: (warning) variable `T' redefines formal in enclosing scope
ToolBus: execution stopped due to the above error(s)
