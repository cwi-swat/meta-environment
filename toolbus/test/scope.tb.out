scope.tb, line 7: double declaration of let variable `R'
scope.tb, line 7: (warning) variable `R' redefines let variable in enclosing scope
scope.tb, line 7: (warning) variable `R' redefines let variable in enclosing scope
scope.tb, line 7: (warning) variable `T' redefines formal in enclosing scope
ToolBus: execution stopped due to the above error(s)
